### PR TITLE
allPositive Arrays

### PR DESCRIPTION
**Pull request by anarayan-star**
Could you check my code please? I attempted to learn how to use for loops with if/then statements.

@jfarmer @terrataylor

### CODE REVIEW

File: exercises/arrays/allPositive/allPositive.js [hunk 1]
```diff
@@ -12,14 +12,37 @@
...
+var ashwin = [-123,21312,43241]
+var anusha = [23,4,5,5]
+var work = [0,2,55,23,45,234,56,23,56]
 if (require.main === module) {
   console.log('Running sanity checks for allPositive:');
 
+  var val = allPositive(base_array);
+  var val2 = allPositive(ashwin); 
+  var val3 = allPositive(anusha);
+  var val4 = allPositive(work)
```
Comment (terrataylor) [inline exercises/arrays/allPositive/allPositive.js:43]: Make sure that you console log all of your sanity checks to make sure that you are getting the correct results of your function calls

File: exercises/arrays/allPositive/allPositive.js [hunk 2]
```diff
@@ -12,14 +12,37 @@
  *   and false otherwise.
  */
 function allPositive(array) {
+  for (let index = 0; index < array.length; index++) {
+    if (array[index] <= 0) {
+      return false;
+    }
+  }
+  return true;
```
Comment (terrataylor) [inline exercises/arrays/allPositive/allPositive.js:20]: Great use of the for loop and return statements.  You may want to double check your if statement.  Is zero considered to be negative?

Comment (anarayan-star) [reply exercises/arrays/allPositive/allPositive.js:20]: Thank you! I believe in the example, 0 was considered to be negative or not positive

Comment (jfarmer) [reply exercises/arrays/allPositive/allPositive.js:20]: `0` is neither positive nor negative.

So "Are any negative?" is a different question than "Are all positive?"

For example, `[0, 0, 1]` has no negative elements, but it has two non-positive elements.

Comment (anarayan-star) [reply exercises/arrays/allPositive/allPositive.js:20]: Right, so the output would still be false right?

Comment (jfarmer) [reply exercises/arrays/allPositive/allPositive.js:20]: Yep, `allPositive([0, 0, 1])` should return `false` (and yours does).

Comment (terrataylor) [reply exercises/arrays/allPositive/allPositive.js:20]: Agreed, sorry for the confusion, my bad math :)

**Pull request closed without merging**